src/hooks/api/getters/useApiTokens/useApiTokens.ts

```diff
@@ -0,0 +1,33 @@
+import useSWR, { mutate } from 'swr';
+import { useState, useEffect } from 'react';
+import { formatApiPath } from '../../../../utils/format-path';
+
+const useApiTokens = () => {
+    const fetcher = async () => {
+        const path = formatApiPath(`api/admin/api-tokens`);
+        const res = await fetch(path, {
+            method: 'GET',
+        });
+        return res.json();
+    };
+
+    const { data, error } = useSWR(`api/admin/api-tokens`, fetcher);
```

Comment:
Just a minor detail, but extracting the key into a API_TOKENS_CACHE_KEY constant makes it less error prone, and allows us to export the key.

Reply:
thanks 👍🏼